app/views/admin/partners/index.html.erb

```diff
@@ -32,7 +32,7 @@
                   <%= link_to partner.name, partner -%>
                 </td>
                 <td> <%= partner.code -%> </td>
-                <td> <%= link_to 'Link', partner.privacy_url -%> </td>
+                <td> <%= link_to 'Link', partner.privacy_url if partner.privacy_url-%> </td>
```

Comment:
Can we check for partner.privacy_url.present? here? Or, otherwise, normalize the field to be non-empty. Partner#privacy_url can end up as "" as a result of editing.